### DISCUSSION TITLE
Hide Content for all users except for the target user

### DISCUSSION THREAD

**Discussion by Suman2023** [eternnoir/pyTelegramBotAPI#2055] [Q&A] [answered]
So i want to reply using a bot in a group with condition that the reply can only be seen by the user i am replying to. I found a doc and tried following it. But it seems to not behave how the comment was written in the code.

I have attached both the code snippets and the comment in the function

# Param comment on the library

:param protect_content: If True, the message content will be hidden for all users except for the target user
:type protect_content: :obj:`bool`

# My Code Snippet
@bot.message_handler(commands=["help"])
def help_me(message):
    current_user = message.from_user.username
    # user_id = message.from_user.id
    bot.reply_to(
        message,
        get_help_message(current_user=current_user),
        reply_markup=inline_keyboard_markup,
        parse_mode="Markdown",
        protect_content=True,
    )

Whenever the bot replies to the command everyone in the group can see it!
PLEASE HELP ME OUT. THIS IS FOR A CLASS WHERE I AM TEACHING STUDENTS 🙏

**Comment by Badiboy** [comment]
Group is the place where EVERYONE will see EVERYTHING. You cannot make private messages in the group.

**Comment by Badiboy** [comment, marked as answer]
Comment is wrong. It should be:

> protect_content | Boolean | Optional | Protects the contents of the sent message from forwarding and saving

we'll fix it.

**Reply by Suman2023** [reply]
Thanks. I felt this was an awesome feature though for bot exclusive as it would have made the command feature be more valuable feature as new members could go for /help and you don't want to fill the group chat with the same message. Any way Thank You @Badiboy it seems to be fixed on master.
